### PR TITLE
CAD-508: analysis updates

### PR DESCRIPTION
**Pull request by deepfire**
1. Import legacy Tx throughput analysis scripts
1. update `xbk.sh` to use the new `BlockFetch` trace -- both in content and format/structure
1. some basic amenities:  `Makefile` and some git ignores
1. update `xtx.sh` to structured logging & use shorter `GenTxIds`

Tested on a slightly-tweaked `cluster3nodes`.

# TODO

- [ ] depends on https://github.com/input-output-hk/cardano-node/pull/668
- [ ] Test in real deployment.

### CODE REVIEW

File: benchmarks/cluster3nodes/configuration/config-node-explorer.yaml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+# if wanted, the GUI is listening on this port:
+#hasGUI: 12871
+
+# if wanted, the EKG interface is listening on this port:
+#hasEKG: 12781
+
+# here we set up outputs of logging in 'katip':
+setupScribes:
+  - scKind: FileSK
+    scName: "logs/node-0.json"
```
Comment (CodiePP) [inline benchmarks/cluster3nodes/configuration/config-node-explorer.yaml]: wouldn't this overwrite another node's log files?

File: benchmarks/cluster3nodes/configuration/config-node-explorer.yaml [hunk 2]
```diff
@@ -0,0 +1,174 @@
+# global filter; messages must have at least this severity to pass:
```
Comment (CodiePP) [inline benchmarks/cluster3nodes/configuration/config-node-explorer.yaml]: this configuration is not necessary. please, remove it.

File: scripts/grep-forge.sh [hunk 3]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
```
Comment (CodiePP) [inline scripts/grep-forge.sh]: this script is an old artefact. please, remove it.

File: scripts/grep-switched.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```
Comment (CodiePP) [inline scripts/grep-switched.sh]: this script is an old artefact. please, remove it.

File: scripts/grep-forge.sh (hunk 3 above)
Comment (deepfire) [reply scripts/grep-forge.sh]: Done.

File: benchmarks/cluster3nodes/configuration/config-node-explorer.yaml (hunk 2 above)
Comment (deepfire) [reply benchmarks/cluster3nodes/configuration/config-node-explorer.yaml]: Done.

File: scripts/grep-switched.sh (hunk 4 above)
Comment (deepfire) [reply scripts/grep-switched.sh]: Done.

File: scripts/xtx.sh [hunk 5]
```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
```
Comment (CodiePP) [inline scripts/xtx.sh:1]: this should be `jq` script.

**Review by CodiePP — CHANGES_REQUESTED** [review]

File: benchmarks/cluster3nodes/configuration/config-node-explorer.yaml (hunk 2 above)
Comment (deepfire) [reply benchmarks/cluster3nodes/configuration/config-node-explorer.yaml]: Done.

File: scripts/xtx.sh (hunk 5 above)
Comment (deepfire) [reply scripts/xtx.sh]: Done.

Comment (deepfire) [reply scripts/xtx.sh:1]: Done.

**Review by CodiePP — APPROVED** [review]
:+1:

**Pull request merged**